### PR TITLE
Added lang.nl

### PR DESCRIPTION
**Pull request by Martijnpold**
Added dutch translation. :)

### CODE REVIEW

File: lang.nl.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*------------------
+Language: English
```
Comment (miyurusankalpa) [inline lang.nl.php]: can you change the language comment to Dutch

**Comment by miyurusankalpa** [comment]
can you make that simple change? thanks btw 👍

File: lang.nl.php (hunk 1 above)
Comment (Martijnpold) [inline lang.nl.php]: Oh, sorry, will do :)

**Comment by miyurusankalpa** [comment]
merged, thanks 👍
have a great hacktoberfest 😃

**Comment by Martijnpold** [comment]
No problem :)
Have fun with the translations :P

**Pull request merged**